numba/tests/test_entrypoints.py

```diff
@@ -31,45 +37,38 @@ def test_init_entrypoint(self):
         # loosely based on Pandas test from:
         #   https://github.com/pandas-dev/pandas/pull/27488
 
-        # FIXME: Python 2 workaround because nonlocal doesn't exist
-        counters = {'init': 0}
-
-        def init_function():
-            counters['init'] += 1
-
-        mod = types.ModuleType("_test_numba_extension")
-        mod.init_func = init_function
+        mod = mock.Mock(__name__='_test_numba_extension')
 
         try:
             # will remove this module at the end of the test
             sys.modules[mod.__name__] = mod
 
             # We are registering an entry point using the "numba" package
-            # ("distribution" in pkg_resources-speak) itself, though these are
+            # ("distribution" in importlib-speak) itself, though these are
             # normally registered by other packages.
```

Comment:
I think this comment is now irrelevant; `distributions` are no longer accessible from the entry point alone, and are not mentioned in the code below.